CONTRIBUTING.md

```diff
@@ -10,72 +10,14 @@ us on [Slack]. Please also take a look at our [code of conduct], which details
 how contributors are expected to conduct themselves as part of the Crossplane
 community.
 
-## Establishing a Development Environment
-
-The Crossplane project consists of several repositories under the crossplane and
-crossplane-contrib GitHub organisations. Most of these projects use the Upbound
-[build submodule]; a library of common Makefiles. Establishing a development
-environment typically requires:
-
-1. Forking and cloning the repository you wish to work on.
-1. Installing development dependencies.
-1. Running `make` to establish the build submodule.
-
-Run `make help` for information on the available Make targets. Useful targets
-include:
-
-* `make test` - Run unit tests.
-* `make e2e` - Run end-to-end tests.
-* `make reviewable` - Run code generation and linters.
-* `make` - Build Crossplane.
-
-Once you've built Crossplane you can deploy it to a Kubernetes cluster of your
-choice. [`kind`] (Kubernetes in Docker) is a good choice for development. The
-`kind.sh` script contains several utilities to deploy and run a development
-build of Crossplane to `kind`:
-
-```bash
-# Build Crossplane locally.
-make
-
-# See what commands are available.
-./cluster/local/kind.sh help
-
-# Start a new kind cluster. Specifying KUBE_IMAGE is optional.
-KUBE_IMAGE=kindest/node:v1.16.15 ./cluster/local/kind.sh up
-
-# Use Helm to deploy the local build of Crossplane.
-./cluster/local/kind.sh helm-install
-
-# Use Helm to upgrade the local build of Crossplane.
-./cluster/local/kind.sh helm-upgrade
-```
-
-When iterating rapidly on a change it can be faster to run Crossplane as a local
-process, rather than as a pod deployed by Helm to your Kubernetes cluster. Use
-Helm to install your local Crossplane build per the above instructions, then:
-
-```bash
-# Stop the Helm-deployed Crossplane pod.
-kubectl -n crossplane-system scale deploy crossplane --replicas=0
-
-# Run Crossplane locally; it should connect to your kind cluster if said cluster
-# is your active kubectl context. You can also go run cmd/crossplane/main.go.
-make run
-```
-
-> Note that local development using minikube and microk8s is also possible.
-> Simply use the `minikube.sh` or `microk8s.sh` variants of the above `kind.sh`
-> script to do so. Their arguments and functionality are identical.
-
 ## Contributing Code
 
 To contribute bug fixes or features to Crossplane:
 
 1. Communicate your intent.
 1. Make your changes.
 1. Test your changes.
-1. Update documentation and examples.
+1. Update documentation.
```

Comment:
We reference this page in the provider-* "Contributing" docs. Because of that. I would like to keep the comment that `examples` should be maintained.